src/Events/Integrations/Modules/Ticket_Emails/Email/RSVP.php

```diff
@@ -133,47 +133,75 @@ public function include_attachments( $attachments, $email_id, $email_class ) {
 	 * @return void
 	 */
 	public function include_event_links( $parent_template ) {
-		$email_class = tribe( RSVP_Email::class );
+		if ( ! $this->should_show_links( $parent_template, $email_class, $args ) ) {
+			return;
+		}
 
-		// Bail early if the email class is not enabled.
-		if ( ! $email_class->is_enabled() ) {
+		$args['event_gcal_link'] = tribe( Google_Calendar::class )->generate_single_url( $args['event']->ID );
+		$args['event_ical_link'] = tribe_get_single_ical_link( $args['event']->ID );
+
+		if ( ! empty( $args['preview'] ) ) {
+			$args['event_gcal_link'] = '#';
+			$args['event_ical_link'] = '#';
```

Comment:
The fix for gcal is because gcal doesn't always return a link. We can do it like this, to overwrite the ical to `#`, but it's not "necessary" as the gcal is.

Reply:
For preview, I didn't want it to cause problems if users click on the link. By using `#`, it won't take them to another page. Just a better user experience, I feel.